src/pkg/backup/details/sharepoint.go

```diff
@@ -58,7 +61,7 @@ func (i SharePointInfo) Headers() []string {
 	case SharePointLibrary:
 		return []string{"ItemName", "Library", "ParentPath", "Size", "Owner", "Created", "Modified"}
 	case SharePointList:
-		return []string{"ListName", "ListItemsCount", "Owner", "Created", "Modified"}
+		return []string{"ListName", "ListItemsCount", "SiteURL", "Template", "Owner", "Created", "Modified"}
```

Comment:
Can you post an example of what the `Template` is going to look like when written to the CLI in this tabular output?  Templates are usually not human-friendly.  I'm dubious about whether we'll want to show it at all.

In general, when updating any CLI display, it's helpful if you add one or more example outputs in comments within the PR so that reviewers can see what the changes will look like on the user's end.

Reply:
@ryanfkeepers 
Here is how it would look like. We need `Template` to either not backup `` template lists or not restore it. Becaue it gives `Invalid template used` while creating with that template. I belive it is system generated and not for users to create and those lists are not even visible in the site.

```
 ID            List                                                    Items  SiteURL                                                    Template         Created               Modified            
  091a4e6cb4f8  Corso_Restore_14-Dec-2023_17-37-40_Events               0      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  events           2023-12-14T17:37:58Z  2023-12-14T17:38:01Z
  d623f7f6a8e9  Events                                                  0      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  events           2023-12-10T06:10:56Z  2023-12-13T17:19:57Z
  7ef15e85ea25  Corso_Restore_14-Dec-2023_17-37-40_Documents            0      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  documentLibrary  2023-12-14T17:37:56Z  2023-12-14T17:37:57Z
  9b8a6eed824a  Corso_Restore_14-Dec-2023_17-37-40_list-with-yes-no     2      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  genericList      2023-12-14T17:38:15Z  2023-12-14T17:38:16Z
  e52ef055db79  list-with-datetime                                      1      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  genericList      2023-12-13T18:40:03Z  2023-12-14T07:30:29Z
  d09086fef4e7  Documents                                               0      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  documentLibrary  2023-12-10T06:10:54Z  2023-12-10T06:10:54Z
  ce68f43d5cf1  Corso_Restore_14-Dec-2023_17-37-40_list-with-multiline  1      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  genericList      2023-12-14T17:38:11Z  2023-12-14T17:38:11Z
  ed98d30966d0  new-books                                               2      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  genericList      2023-12-13T09:23:06Z  2023-12-14T07:30:29Z
  719c04a93e14  list-with-yes-no                                        2      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  genericList      2023-12-13T18:34:52Z  2023-12-14T07:30:35Z
  0a5afc266a59  Corso_Restore_14-Dec-2023_17-37-40_list-with-metadata   1      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  genericList      2023-12-14T17:38:09Z  2023-12-14T17:38:10Z
  6ff9af316541  link-with-choice                                        1      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  genericList      2023-12-13T11:56:52Z  2023-12-14T07:30:29Z
  bd9a5e592ca6  list-with-multiline                                     1      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  genericList      2023-12-13T18:41:51Z  2023-12-14T07:30:29Z
  4eaa2bbf521f  list-with-lookup                                        1      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  genericList      2023-12-13T18:44:56Z  2023-12-14T07:30:29Z
  5e4f541906db  list-with-metadata                                      1      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  genericList      2023-12-13T18:46:17Z  2023-12-14T07:30:30Z
  28d9be63ff0d  Corso_Restore_14-Dec-2023_17-37-40_list-with-person     1      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  genericList      2023-12-14T17:38:13Z  2023-12-14T17:38:14Z
  b96b04147533  Corso_Restore_14-Dec-2023_17-37-40_new-books            2      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  genericList      2023-12-14T17:38:17Z  2023-12-14T17:38:19Z
  8738089bb528  Corso_Restore_14-Dec-2023_17-37-40_list-with-datetime   1      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  genericList      2023-12-14T17:38:05Z  2023-12-14T17:38:05Z
  49940bd33ed7  Corso_Restore_14-Dec-2023_17-37-40_list-with-lookup     1      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  genericList      2023-12-14T17:38:07Z  2023-12-14T17:38:08Z
  915f535a99dc  Corso_Restore_14-Dec-2023_17-37-40_link-with-choice     1      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  genericList      2023-12-14T17:38:03Z  2023-12-14T17:38:03Z
  7276fbf86223  list-with-person                                        1      https://10rqc2.sharepoint.com/sites/site-4754-small-lists  genericList      2023-12-13T12:19:52Z  2023-12-14T07:30:33Z

```

Reply:
> We need Template to either not backup webTemplateExtensionsList template lists or not restore it.

I grok.  But I don't think we should display it.  That means the end user needs to know the difference between backup items that are, and are not, restorable.  It pushes complexity out onto the user, instead of handling it ourselves, which isn't up to our standards.

What you should do, instead, is to filter all non-restorable template types out of the backup details list.  That will ensure end users are able to restore any item we display.  If we want to view the unfiltered list, then passing in the `--skip-reduce` string should display all list items without filtering them.

That way we can keep the template in the details info, but not display it to the end user.

Reply:
I will implement `--skip-reduce` for backup details display in a different PR.
And filter `webTemplateExtensionsList` lists while restoring.